ui/ui-components/pages/index.tsx

```diff
@@ -53,6 +53,7 @@ export default function Page(props) {
               <Button
                 variant="primary"
                 size="lg"
+                className="signIn"
```

Comment:
For the other buttons you added an `id`, but this one you added a `class`, why the change?  

Usually, classes are for style or type (eg: this is a "submit" or "cancel" type button that looks a certain way) an IDs are to identity the specific instance of the button (eg: add-to-cart vs sign-in) both of which might be of class "submit".

Reply:
True.  There are two sign in buttons in the app and I had thought it poor form to use an id for something with more than one instance.

Also, the classes and IDs aren't explicitly needed for testing purposes anymore as the CTA button is now button-outline instead of button-primary, so they could also just be removed all around if that seems better.

Reply:
removed the unneeded css ids/classes.